### PR TITLE
[custom policies] Put UI under rolling update

### PR DESCRIPTION
**Pull request by didierofrivia**
We could revert this PR when Custom Policies are good to deliver

### CODE REVIEW

**Comment by guicassolato** [comment]
If it's always `false`, why not just deleting the render command and later we can simply revert this PR?

**Comment by didierofrivia** [comment]
@thomasmaas it worked because it has another RollingUpdate going on.
@guicassolato yup, it also makes sense

File: app/controllers/provider/admin/registry/policies_controller.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def policy_params
   end
 
   def authorize_policies
-    authorize! :manage, :policy_registry
+    authorize! :manage, :policy_registry_ui
```
Comment (Martouta) [inline app/controllers/provider/admin/registry/policies_controller.rb]: I don't know if it should have both or only this one for the real usage later on, but right now it is going to be false anyway so for now it is ok 👍

Comment (hallelujah) [reply app/controllers/provider/admin/registry/policies_controller.rb]: See the rolling update, it is inheriting from `:policy_registry`

Comment (Martouta) [reply app/controllers/provider/admin/registry/policies_controller.rb]: ah right sorry, i had not noticed. Good one 👍

File: app/lib/logic/rolling_updates.rb [hunk 2]
```diff
@@ -287,6 +287,12 @@ def missing_config
         end
       end
 
+      class PolicyRegistryUi < PolicyRegistry
+        def enabled?
+          super && provider_can_use?(:policy_registry)
```
Comment (guicassolato) [inline app/lib/logic/rolling_updates.rb]: 3-level rolling update feature. I think I opened Pandora's box :)

File: test/integration/provider/registry/policies_controller_test.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ def setup
     @provider = FactoryBot.create(:provider_account)
     login! @provider
     rolling_updates_off
-    rolling_update(:policy_registry, enabled: true)
+    rolling_update(:policy_registry_ui, enabled: true)
```
Comment (guicassolato) [inline test/integration/provider/registry/policies_controller_test.rb]: Do we need a test for `policy_registry_ui : false`?

Comment (hallelujah) [reply test/integration/provider/registry/policies_controller_test.rb]: Well, if you remove it you will see tests failing. I do not know ...
This is just "renaming" or putting in another level the UI. So if it was not done in the first PR I think it was for a reason

File: app/lib/logic/rolling_updates.rb (hunk 2 above)
Comment (hallelujah) [reply app/lib/logic/rolling_updates.rb]: Right 🤣 I was thinking the same but I do not see any other solution where you say you need `:policy` and  `:policy_registry`

**Pull request merged**